app/Services/SearchService.php

```diff
@@ -259,6 +259,21 @@ public function find()
                 'alias_id' => $model->alias_id, // @phpstan-ignore-line
                 'advanced_mention' => Mentions::advancedMentionHelper($model->name),
             ];
+
+            //If the result is a map, also add its explore page as a result.
+            if ($model->type() == 'map' && $model->child->explorable()) {
```

Comment:
It's now showing up in the @mention system, which isn't needed. It's the same endpoint, but we have a param that separates both.
![image](https://user-images.githubusercontent.com/3263440/199125364-a231a1a1-a3b1-463f-9e9c-15a34a54f6da.png)